packages/fluentui/docs/src/components/ComponentDoc/ComponentControls/ComponentControls.tsx

```diff
@@ -79,95 +79,97 @@ const ComponentControls: React.FC<ComponentControlsProps> = props => {
-                ),
-              },
-              {
-                icon: <LinkIcon style={{ width: '20px', height: '20px' }} />,
-                children: (Component, props) => (
-                  <CopyToClipboard key="copy-link" value={anchorName}>
-                    {(active, onClick) => (
-                      <Tooltip
-                        content={active ? 'Copied!' : 'Permalink'}
-                        trigger={
-                          <Component
-                            {...props}
-                            onClick={(e: React.SyntheticEvent) => {
-                              onClick();
-                              onCopyLink(e);
-                            }}
-                          />
-                        }
-                      />
-                    )}
-                  </CopyToClipboard>
-                ),
-              },
-            ]}
+                {
+                  icon: <OpenOutsideIcon style={{ width: '20px', height: '20px' }} />,
+                  children: (Component, props) => (
+                    <Tooltip content="Popout" key="maximize" trigger={<Component {...props} />} />
+                  ),
+                  as: NavLink,
+                  to: `/maximize/${_.kebabCase(
+                    examplePath
+                      .split('/')
+                      .slice(-1)
+                      .pop(),
+                  )}/${showRtl}`,
+                  target: '_blank',
+                  rel: 'noopener noreferrer',
+                },
+                {
+                  key: 'divider-2',
+                  style: { margin: '0 5px' },
+                  kind: 'divider',
+                },
+                {
+                  onClick: onCodeSandboxClick,
+                  icon: codeSandboxIcon,
+                  children: (Component, props) => (
+                    <Tooltip content={codeSandboxTooltip} key="show-codesandbox" trigger={<Component {...props} />} />
+                  ),
+                },
+                {
+                  icon: <LinkIcon style={{ width: '20px', height: '20px' }} />,
+                  children: (Component, props) => (
+                    <CopyToClipboard key="copy-link" value={anchorName}>
+                      {(active, onClick) => (
+                        <Tooltip
+                          content={active ? 'Copied!' : 'Permalink'}
+                          trigger={
+                            <Component
+                              {...props}
+                              onClick={(e: React.SyntheticEvent) => {
+                                onClick();
+                                onCopyLink(e);
+                              }}
+                            />
+                          }
+                        />
+                      )}
+                    </CopyToClipboard>
+                  ),
+                },
+              ] as MenuItemProps[]
```

Comment:
Let's avoid this cast, let's extract this array to `items` variable and define it as `const items: Menu['items'] = []`

Reply:
Done